src/main/java/com/minecolonies/coremod/util/Utils.java

```diff
@@ -3,11 +3,6 @@
 import com.minecolonies.compatibility.Compatibility;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.minecolonies.compatibility.Compatibility'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)